clients/drcachesim/analyzer.cpp

```diff
@@ -122,6 +122,15 @@ analyzer_t::record_is_timestamp(const memref_t &record)
         record.marker.marker_type == TRACE_MARKER_TYPE_TIMESTAMP;
 }
 
+template <>
+void
+analyzer_t::set_wait_record(memref_t &record)
```

Comment:
create_wait_marker that returns the memref_t would be more readable.

Reply:
Done.